### PR TITLE
[AVKit] Update to Xcode 9 beta 1

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: src/avkit.cs [hunk 1]
```diff
@@ -425,6 +488,22 @@ interface AVKitMetadataIdentifier {
 		NSString ExternalUserProfileIdentifier { get; }
 		[Field ("AVKitMetadataIdentifierPlaybackProgress")]
 		NSString PlaybackProgress { get; }
+
+		[TV (11,0)]
```
Comment (spouliot) [inline src/avkit.cs]: Is that under a tvOS only define ?

Comment (dalexsoto) [inline src/avkit.cs]: Yup

```
	[Static]
	[NoiOS, TV (10,1), NoWatch, NoMac]
	interface AVKitMetadataIdentifier {
```

Comment (spouliot) [inline src/avkit.cs]: 👍

File: src/avkit.cs [hunk 2]
```diff
@@ -134,6 +139,16 @@ interface AVPlayerViewController {
 		[Export ("updatesNowPlayingInfoCenter")]
 		bool UpdatesNowPlayingInfoCenter { get; set; }
 
+		[iOS (11, 0)]
```
Comment (VincentDondain) [inline src/avkit.cs]: Minor comment `[iOS (11,0)]` is prettier (note: you're sometimes not adding a space in this diff).

**Pull request merged**